.eslintignore

```diff
@@ -0,0 +1,9 @@
+node_modules
+node_modules/**/*
+.DS_Store
+tmp/**/*
+.idea
+.idea/**/*
+*.iml
+*.log
+coverage
```

Comment:
FWIW, only `coverage/` is needed in this file

Reply:
noted. will change